css/step1-atom.css

```
@@ -35,26 +21,15 @@
   letter-spacing: 3px;
   text-transform: uppercase;
   color: #fff;
```

```suggestion
  color: #fff;
  display: inline-block;
```

1.  As the previous comment, please visually check your work in the web browser.  This atom does not display properly without `display: inline-block`.  It's ok to use `display` on atoms if needed.  On the other hand you rarely want to use `position` in atoms since it is related to layout.
2. What is the default `display` for `<a>` tags?  `display: inline`, `display: inline-block`, or `display: block`?  In other words, if you don't specify `display` style for an `<a>` tag what is the default?
3. What is the difference between `display:inline`, `display: inline-block`, and `display: inline-block`?  Please explain.